apps/mull-ui/src/app/pages/create-event/create-event.scss

```diff
@@ -15,6 +15,11 @@
   .custom-time-picker-container {
     margin: 0 0 1.25rem;
   }
+
+  .date-calendar-container,
+  .custom-text-input-container {
+    width: 100%;
+  }
```

Comment:
@r-saba  already applied this styling in his PR, you can remove it when you update your branch with his changes